src/Foundatio/Serializer/MessagePackSerializer.cs

```diff
@@ -6,17 +6,29 @@
 namespace Foundatio.Serializer {
     public class MessagePackSerializer : ISerializer {
         private readonly IFormatterResolver _formatterResolver;
+        private readonly bool _compressEnabled;
 
-        public MessagePackSerializer(IFormatterResolver resolver = null) {
+        public MessagePackSerializer(IFormatterResolver resolver = null, bool compressEnabled = false) {
```

Comment:
should this be called `useCompression` ? or should we have a serializer settings class for this (do we see more options being used here)?